### PR TITLE
Release Plan

### PR DESCRIPTION
**Pull request by dscervantes**
Adding Release Plan to rhtap-hcc-pipeline-tenant

### CODE REVIEW

**Comment by dscervantes** [comment]
@scoheb Release Plan :)

File: auto-generated/cluster/stone-prd-rh01/tenants/rhtap-hcc-pipeline-tenant/appstudio.redhat.com_v1alpha1_releaseplan_insights-ingress-go-release-to-quay-rhtap-ser.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  labels:
+    release.appstudio.openshift.io/auto-release: "true"
+    release.appstudio.openshift.io/standing-attribution: "true"
+  name: insights-ingress-go-release-to-quay-rhtap-ser
+  namespace: rhtap-hcc-pipeline-tenant
+spec:
+  application: insights-ingress-go
+  displayName: insights-ingress-go release-to-quay
+  target: rh-managed-rhtap-ser-tenant
```
Comment (scoheb) [inline auto-generated/cluster/stone-prd-rh01/tenants/rhtap-hcc-pipeline-tenant/appstudio.redhat.com_v1alpha1_releaseplan_insights-ingress-go-release-to-quay-rhtap-ser.yaml:12]: this wrong. you should be releasing to some other managed workspace.

**Comment by dscervantes** [comment]
@scoheb is rhtap-releng-tenant for insights?

**Comment by scoheb** [comment]
> @scoheb is rhtap-releng-tenant for insights?

Yes. now it is coming back to me...

the target should be **rhtap-releng-tenant**

You also need to add the related config to https://gitlab.cee.redhat.com/releng/rhtap-release-data following the README

In addition, you can use this as a template: https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/blob/main/config/ReleasePlanAdmission/trusted-content/exhort.yaml?ref_type=heads

You are targeting to release to a quay repo correct? and not registry.redhat.io?

**Comment by dscervantes** [comment]
@scoheb Let me do the releng stuff first, and yes to the quay repo !

**Comment by dscervantes** [comment]
@scoheb waiting to see if this is correct https://gitlab.cee.redhat.com/releng/rhtap-release-data/-/merge_requests/100

**Comment by dscervantes** [comment]
@scoheb This should have the correct target now that MR 100 was merged

File: cluster/stone-prd-rh01/tenants/rhtap-hcc-pipeline-tenant/release_plan.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: insights-ingress-go-release-to-quay-rhtap-ser
+  namespace: rhtap-hcc-pipeline-tenant
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
+    release.appstudio.openshift.io/standing-attribution: 'true'
+spec:
+  application: insights-ingress-go
+  displayName: insights-ingress-go release-to-quay
+  target: rhtap-releng-tenant
```
Comment (scoheb) [inline cluster/stone-prd-rh01/tenants/rhtap-hcc-pipeline-tenant/release_plan.yaml:12]: missing new line

**Review by scoheb — APPROVED** [review]

**Pull request merged**